### PR TITLE
Support app bundles in default apps

### PR DESCRIPTION
**Pull request by QuentinBisson**
Towards https://github.com/giantswarm/giantswarm/issues/23425

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by QuentinBisson** [comment]
@giantswarm/team-honeybadger can you check this PR?

File: service/controller/key/key.go [hunk 1]
```diff
@@ -72,6 +73,10 @@ func DNSIP(clusterIPRange string) (string, error) {
 	return ip.String(), nil
 }
 
+func IsBundle(appName string) bool {
+	return strings.HasSuffix(appName, "-bundle")
```
Comment (kubasobon) [inline service/controller/key/key.go:77]: How does that work for `security-pack`, for example?

Comment (QuentinBisson) [reply service/controller/key/key.go:77]: It does not and it should be renamed to security-bundle https://intranet.giantswarm.io/docs/product/pdr/008_app_bundle_naming/

File: service/controller/resource/app/create.go [hunk 2]
```diff
@@ -158,6 +158,17 @@ func (r *Resource) newApp(ctx context.Context, cr capi.Cluster, appSpec AppSpec)
 		kubeConfig = v1alpha1.AppSpecKubeConfig{
 			InCluster: true,
 		}
+	} else if key.IsBundle(appSpec.App) {
+		kubeConfig = v1alpha1.AppSpecKubeConfig{
+			InCluster: true,
+			Context: v1alpha1.AppSpecKubeConfigContext{
```
Comment (ljakimczuk) [inline service/controller/resource/app/create.go:164]: Do we need  this information, as well as the secret name, when we have in-cluster app? The bundle App CR is installed in the Management Cluster context.

Comment (QuentinBisson) [reply service/controller/resource/app/create.go:164]: Maybe not but this is how it's done for the security packs so I did the same. Let me correct it

Comment (QuentinBisson) [reply service/controller/resource/app/create.go]: Fixed in latest commig

**Review by ljakimczuk — APPROVED** [review]

**Pull request merged**